packages/utils/index.js

```diff
@@ -1,26 +1,22 @@
 // @flow
 
 // Helpers
-export { capitalize } from './helpers/capitalize';
+export { Capitalize } from './helpers/Capitalize';
```

Comment:
Maybe `Capitalize` can be also moved to `formatters`.

Reply:
Sure! that way we get can get rid of the helpers